<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainForm</class>
 <widget class="QWidget" name="QMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procesamiento de im√°genes</string>
  </property>
  <widget class="QFrame" name="frameOrig">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameDest">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>40</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCargar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSalir">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrig">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Imagen Origen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDest">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Imagen Destino</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCopiarO">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Copiar origen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCopiarD">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Copiar destino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVoltearV">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltear Vertical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGirarI">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Girar Izquierda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGirarD">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Girar Derecha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGuardar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QFrame" name="options">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>841</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonNegar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Negar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVoltearH">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltear Horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRecortar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Recortar</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUmbralizar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Umbralizar</string>
    </property>
   </widget>
  </widget>
  <zorder>options</zorder>
  <zorder>frameOrig</zorder>
  <zorder>frameDest</zorder>
  <zorder>pushButtonCargar</zorder>
  <zorder>pushButtonSalir</zorder>
  <zorder>labelOrig</zorder>
  <zorder>labelDest</zorder>
  <zorder>pushButtonCopiarO</zorder>
  <zorder>pushButtonCopiarD</zorder>
  <zorder>pushButtonVoltearV</zorder>
  <zorder>pushButtonGirarI</zorder>
  <zorder>pushButtonGirarD</zorder>
  <zorder>pushButtonGuardar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
